Source/SpacePark/Menu.cs

```diff
@@ -7,55 +7,61 @@ namespace SpacePark
 {
     public class Menu
```

Comment:
Nice to see that you separated the menu into its own class!  (also nice logo 👌 )